### DISCUSSION TITLE
Why does a simulation considering only adhesion converge slower and give weird results, as compared to a simulation considering both adhesion and plasticity?

### DISCUSSION THREAD

**Discussion by sitangshugk95** [ContactEngineering/Adhesion#77] [Q&A]
I am trying to simulate the evolution in contact area for different displacements for a [rough surface](https://doi.org/10.57703/ce-b9zfz) for three different scenarios:

- **Only elastic deformation**

For this, I am using the [Hardwall simulation example](https://github.com/ContactEngineering/ContactMechanics/blob/master/examples/Hardwall_Simulation.ipynb) in Contact Mechanics and getting results that make sense: 
![image](https://user-images.githubusercontent.com/108830413/211443917-0c578b03-997f-4ebd-a81f-e75aa478a8dd.png)

- **Elastic + adhesion + plastic deformation**

For this, I am using the [Adhesive plastic cone example](https://github.com/ContactEngineering/ContactMechanics/blob/master/examples/Hardwall_Simulation.ipynb) in Adhesion and getting results qualitatively that make sense (more contact area than 1):
![image](https://user-images.githubusercontent.com/108830413/211444057-345a1199-e3fe-4eee-bbe3-a513b9f35aab.png)

- **Elastic + adhesion only (ignoring plasticity effect)**

For this, I use the same code as before, but replace `PlasticSmoothContactSystem` by `SmoothContactSystem`. However, the convergence takes significantly longer, and gives results that does not make sense (i.e. contact only along the periphery, very high pressure values as visible in the colorbar autorange):
![image](https://user-images.githubusercontent.com/108830413/211445255-91caeeb8-308c-4c25-bd9b-f741dffd8305.png)

Questions:

1. _Is there a reason why the adhesion+plastic simulation converges faster than the simulation which has ONLY adhesion?_ It seems non-intuitive since ignoring plasticity might make the system simpler, but I might be wrong.
2. _What might be causing this contacts to be formed only on the periphery?_ The contact regions look very weird.

**Comment by sitangshugk95** [comment]
@sannant

**Reply by sannant** [reply]
Difficult to tell. 

1. Are you sure the simulation is converged? 
2. Are you sure about the paramtrisation ? Try starting with weaker and longer-ranged adhesion. Example of parametrisations that correspond to rather weak adhesion: https://link.springer.com/article/10.1007/s11249-021-01454-6
3.

**Reply by sitangshugk95** [reply]
Thank you for your inputs @sannant .

1. What is the general procedure for ensuring that the simulation is converged? 
Is it by seeing if the contact areas are remaining same by changing the following: 
(A) increasing the force/penetration tolerance? OR 
(B) increasing the maximum number of iterations? OR
(C) increasing the number of data points per um of the surface? OR
(D) any other way I am not aware of? 
_(Generally for FEM models I would make the mesh finer but here I am kind of restricted by the AFM scan already taken. Ofcourse I can take a new scan, but I just wanted to get your feedback on the order of preference, based on your experience)_

2. When you say parametrization, do you mean the interaction law being used? (Eg; use cubic DMT-like interaction, ias in the Contact Mechanics challenge, instead of the vdW82)? 
(_Just wanting to make sure I'm understanding parametrization correctly_)

**Reply by sitangshugk95** [reply]
> Thank you for your inputs @sannant .
> 
> 1. What is the general procedure for ensuring that the simulation is converged?
>    Is it by seeing if the contact areas are remaining same by changing the following:
>    (A) increasing the force/penetration tolerance? OR
>    (B) increasing the maximum number of iterations? OR
>    (C) increasing the number of data points per um of the surface? OR
>    (D) any other way I am not aware of?
>    _(Generally for FEM models I would make the mesh finer but here I am kind of restricted by the AFM scan already taken. Ofcourse I can take a new scan, but I just wanted to get your feedback on the order of preference, based on your experience)_
> 2. When you say parametrization, do you mean the interaction law being used? (Eg; use cubic DMT-like interaction, as in the Contact Mechanics challenge, instead of the vdW82)?
>    (_Just wanting to make sure I'm understanding parametrization correctly_)

@sannant do you have any suggestions for these? I am planning to do a benchmark test on the Contact Mechanics challenge - is that potential defined in any of the Interactions in the package?
